backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -0,0 +1,81 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Squirrel.Core.Common.DTO.Projects;
+using Squirrel.Core.BLL.Interfaces;
+using Squirrel.Core.BLL.Services.Abstract;
+using Squirrel.Core.Common.Exceptions;
+using Squirrel.Core.DAL.Context;
+using Squirrel.Core.DAL.Entities;
+
+namespace Squirrel.Core.BLL.Services
+{
+    public class ProjectService : BaseService, IProjectService
+    {
+        public ProjectService(SquirrelCoreContext context, IMapper mapper): base(context, mapper){ }
+
+        public async Task<ProjectDto> AddProjectAsync(ProjectDto projectDto)
+        {
+            var projectEntity = _mapper.Map<Project>(projectDto);
+            
+            projectEntity.DbEngine = projectDto.DbEngine;
```

Comment:
Why you can't make it in mapper?

Reply:
> Why you can't make it in mapper?
It was an accident, I'll clean it up now